.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,17 +1,22 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-title: "[IDEA] "
+title: '[IDEA] '
 labels: idea
 assignees: ''
-
 ---
 
-**Is your feature request related to a problem?**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+## Problem to solve
+
+<!-- Is your feature request related to a problem?
```

Comment:
Not sure I understand why these are commented out?

Reply:
It might be personal preference 😂 . When the contributor is editing their PR description, they'll be able to see these comments, but they won't get displayed once the submit the comment.

To minimize the impact of this PR, I'll uncomment them 👍

Reply:
@peripheralist addressed in https://github.com/jbx-protocol/juice-interface/pull/275/commits/c1e938934d55315d80dafed597f279af8ed5ba57, back to you 🏓

Reply:
ahh, nice trick! @aeolianeth does the `<!--` still show up in the message text when editing?

Reply:
@peripheralist yup. But I actually think having them not-commented might be better, adds a bit more pressure for someone to write a more complete description 😉